oh.sh

```diff
@@ -43,14 +43,14 @@ WRITE_CONFIG_FILES="off"
 ############## OH general configuration - change at your own risk :-) ##############
 
 # OH_PATH is the directory where Open Hospital files are located
-# OH_PATH=/usr/local/OpenHospital/oh-1.13
+# OH_PATH=/usr/local/OpenHospital/oh
 
 # set OH mode to PORTABLE | CLIENT | SERVER - default set to PORTABLE
 #OH_MODE="PORTABLE" 
 
 # language setting - default set to en
-OH_LANGUAGE_LIST="en|fr|es|it|pt|ar"
-#OH_LANGUAGE="en" # default
+OH_LANGUAGE_LIST=("ar" "de" "en" "es" "fr" "it" "pt" "sq")
+OH_LANGUAGE_LIST_INFO=("arabic" "german" "english" "spanish" "french" "italian" "portuguese" "albanian")
```

Comment:
should we capitalize them?

```suggestion
OH_LANGUAGE_LIST_INFO=("Arabic" "German" "English" "Spanish" "French" "Italian" "Portuguese" "Albanian")
```

Reply:
See https://github.com/informatici/openhospital-gui/pull/1959